test/karma/test-app/slot-ref/cmp.tsx

```diff
@@ -0,0 +1,18 @@
+import { Component, Element, h, Host } from '@stencil/core';
+
+@Component({
+  tag: 'slot-ref',
+  shadow: false,
+  scoped: true
+})
+export class SlotRef {
+  @Element() hostElement: HTMLElement;
+
+  render() {
+    return (
+      <Host>
+        <slot name="title" ref={() => (this.hostElement.setAttribute('data-ref', 'called'))} />
```

Comment:
Can we update the test and use the provided element reference to verify that it was provided in the callback?

Reply:
done 🚀